sinks/dashboard/items/plot_dash_item.py

```
@@ -162,6 +180,12 @@ def on_data_update(self, stream, payload):
                            if self.points[item] else 0 for item in self.series]
             for v in last_values:
                 current_values += f"[{v: < 4.4f}] "
+        if self.parameters.param('Show Slope of Linear Approx.').value():
+            slope_values: list[str] = []
+            for stream in self.series:
```

Pylint (3.3.7)

[refactor] Redefining argument with the local name 'stream'

(R1704)